eli5/_feature_names.py

```diff
@@ -110,6 +117,20 @@ def filtered_by_re(self, feature_re):
             ),
             indices)
 
+    def handle_filter(self, feature_filter, feature_re, x=None):
+        if feature_re is not None and feature_filter:
+            raise TypeError('pass either feature_filter or feature_re')
```

Comment:
Aha, thanks for the catch! I chose TypeError cause it's raised for missing/extra arguments, but ValueError really makes more sense, thanks for the link)